src/linearcombination.jl

```diff
@@ -37,6 +37,18 @@ function Base.:(+)(A₁::LinearMap, A₂::LinearMap)
     return LinearCombination{T}(tuple(A₁, A₂))
 end
 Base.:(-)(A₁::LinearMap, A₂::LinearMap) = +(A₁, -A₂)
+function Base.:(+)(O::ZeroMap, A::LinearMap)
+    size(O) == size(A) || throw(DimensionMismatch("+"))
+    T = promote_type(eltype(O), eltype(A))
```

Comment:
what is the point of computing `T` if then anyway `A` is returned?

Reply:
Good point. This situation here is supposed to occur when the `ZeroMap` was created out of a `CompositeMap` that had a `ZeroMap` factor. So that map does have its own eltype, which probably should not get lost here. So I guess what should be returned is actually `LinearMap{T}(A)`. At least that's what I meant to do originally, but then forgot about it in the next line.